src/Tizen.System.Information/Usage/ProcessMemoryUsage.cs

```diff
@@ -194,12 +294,12 @@ public void Update(IEnumerable<int> pid)
 
             Pids = pid.ToArray<int>();
             IntPtr ptr = new IntPtr();
-            Count = Pids.Count<int>();
+            Count = Pids.Length;
```

Comment:
Fix the following property 
https://github.com/Samsung/TizenFX/blob/d886ee956bc4302582cf3921cd1d857e93b306bf/src/Tizen.System.Information/Usage/ProcessMemoryUsage.cs#L57
to 
```csharp
public int Count => Pids.Length;
```
instead of setting `Count` here.